tests/tools.vitruv.applications.pcmumlclass.tests/src/tools/vitruv/applications/pcmumlclass/tests/PcmUmlclassViewFactory.xtend

```diff
@@ -0,0 +1,88 @@
+package tools.vitruv.applications.pcmumlclass.tests
+
+import tools.vitruv.testutils.TestViewFactory
+import tools.vitruv.framework.views.View
+import org.eclipse.uml2.uml.Model
+import org.eclipse.xtend.lib.annotations.FinalFieldsConstructor
+import org.palladiosimulator.pcm.repository.Repository
+import org.palladiosimulator.pcm.repository.RepositoryComponent
+import org.palladiosimulator.pcm.system.System
+
+@FinalFieldsConstructor
+class PcmUmlclassViewFactory extends TestViewFactory {
+	
+	private def View createUmlView() {
+		createViewOfElements("UML", #{Model})
+	}
+
+	private def View createPcmView() {
+		createViewOfElements("Pcm packages and components", #{Repository, RepositoryComponent})
```

Comment:
PCM stands for Palladio Component Model and should thus also be written capitalised (everywhere in this file).

Reply:
Corrected in e644396f32b0509741a91d19d43f764532a6229a